base/steer/FairRunAna.h

```diff
@@ -100,6 +100,7 @@ class FairRunAna : public FairRun
   private:
     FairRunAna(const FairRunAna& M);
     FairRunAna& operator=(const FairRunAna&) { return *this; }
+    FairEventHeader* fEvtHeader = nullptr;
```

Comment:
Having another thought about this as well:

Could you try to not use member variables, but use local variables in the methods that need `fEvtHeader`?

Like
```cpp
FairRunAna::DoFoo() {
    auto fEvtHeader = GetEventHeader();
    …
    // code that actually needs fEvtHeader
```

that way the relevant methods havr access to the current value and not just an old one?

Reply:
Yes, this is a better way.